backend/src/rtp.rs

```diff
@@ -1618,7 +1674,7 @@ impl Endpoint {
             SequenceNumberReuse::UsedBefore => {
                 trace!("Dropping SRTP packet because we've already seen this seqnum ({}) from this ssrc ({})", seqnum_in_header, header.ssrc);
                 event!("calling.srtp.seqnum_drop.reused");
-                return None;
+                return Err(EndpointError::IncomingSsrcStateNotFound(1));
```

Comment:
The error is incorrect here. It is not "Not Found", it is that the sequence number has been "Used Before". I would create a dedicated error for it. Another issue is that whoever the consumer of the error is, they aren't going to know what 1 or 2 means, better to be explicit here.

Reply:
Yeah @jim-signal  that makes sense, since there is no numbering convention given which would aid in debugging or otherwise be helpful, I will construct a new `SsrcError` type here and let me know if it is valid to be used.